src/components/icon/Icon.tsx

```diff
@@ -52,7 +53,7 @@ const Icon: React.FC<IconProps> = ({
 };
 
 Icon.propTypes = {
-  iconName: PropTypes.string.isRequired,
+  iconName: PropTypes.any.isRequired,
```

Comment:
We should try to avoid using 'any'. I didn't find any good solution for using 'type' with PropTypes.
The only solution is to use:
```
  iconName: PropTypes.string.isRequired as PropTypes.Validator<TkIcon>,
```
With this line, we use at least the 'string' Proptype (and there is no error thanks to the `as PropTypes.Validator<TkIcon>`)